python/tvm/relax/frontend/nn/visitor.py

```diff
@@ -112,6 +113,14 @@ def visit(self, name: str, node: Any) -> Any:
         ret_node: Any
             The new node to replace current node.
         """
+
+        def _leaf_name(parent: str, child: str) -> str:
+            if parent == "":
+                # in the top level of the module
+                return child
+            else:
+                return f"{parent}.{child}"
+
         if isinstance(node, nn.ModuleList):
             for i in range(len(node)):
                 if isinstance(node[i], nn.ModuleList):
```

Comment:
Shall we also change the following parts?

Reply:
It's a good question. I assume the top level is nn.Module, not nn.ModuleList. Hence, when visiting nn.ModuleList, the name should not be left as an empty string. @cyx-6 @junrushao

Reply:
I think it's a reasonable design